week1/homework/src/server.js

```diff
@@ -5,16 +5,40 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
+  function showResponseToClient(response, state) {
+    response.setHeader('Content', 'application/json');
```

Comment:
Really awsome that you moved out this logic from the `switch case`! 🎊  Maybe you can take it one step further and move the fucntion completly out to be outside of the `createServer` function? Also I am missing a status code :) 
Since you are suggesting in your eslint configuration that you use `es6` ,if you feel like taking it even further, you really dont have to, is to make that into an  arrow function.